src/dma.rs

```diff
@@ -563,6 +555,12 @@ impl_target!(
 );
 
 #[cfg(feature = "stm32l0x2")]
+#[cfg(any(
+    feature = "io-STM32L021",
+    feature = "io-STM32L031",
+    feature = "io-STM32L051",
+    feature = "io-STM32L071",
+))]
```

Comment:
Multiple `cfg` attributes are additive (so both need to match), right?

I'm not sure whether `stm32l0x2` is still required, but on the other hand I don't have much experience with DMA yet.